src/features/quiz/results/Results.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button } from "@material-ui/core";
+import React from "react";
+import { BrowserRouter as Router, useHistory } from "react-router-dom";
+import { useAppDispatch, useAppSelector } from "../../../app/hooks";
+import { resetCount, selectCorrectAnswerCount } from "../answers/answerSlice";
+
+const Results = () => {
+  const dispatch = useAppDispatch();
+  const correctAnswerCount = useAppSelector(selectCorrectAnswerCount);
+  const history = useHistory();
+
+  const handleHistory = () => {
```

Comment:
We could rename the function name, could be something else that describe what exactly the function does.

Reply:
Thanks, agree with you. Fixed!